understanding/21/non-text-contrast.html

```diff
@@ -140,7 +140,7 @@ <h4 id="related-focus">Relationship with Focus Visible</h4>
 				<figure id="figure-focus-outer-green">
 					<img src="img/ntc-focus-outer-green.png" alt="Three blue buttons on a white background, the middle has a dark green outline outside of the botton's non-focused boundary." width="400" />
 					<figcaption>
-						The external green indicator (#008000) does contrast with the white background (#FFF) which the component is on, <strong>passing</strong> the criterion. It does not need to contrast with both the component background and the component, as visually the effect is that the button is noticeably larger, and it's not necessary for a user to be able to discern this extra border in isolation. Although this passes non-text contrast, it is not a good indicator unless it is very thick. <span class="wcag22">There is a AAA criterion in WCAG 2.2 that addresses this aspect, <a href="../22/focus-appearance.html">Focus Appearance</a></span>.
+						The external green indicator (#008000) does contrast with the white background (#FFF) which the component is on, <strong>passing</strong> the criterion. It does not need to contrast with both the component background and the component, as visually the effect is that the button is noticeably larger, and it's not necessary for a user to be able to discern this extra border in isolation. Although this passes non-text contrast, it is not a good indicator unless it is very thick. <span class="wcag22">There is a AAA criterion in WCAG 2.2 that addresses this aspect, <a href="focus-appearance.html">Focus Appearance</a></span>.
```

Comment:
(as i think there's some magic xslt or whatever that turns links to SCs etc into proper links ... or are we explicitly trying to link to the understanding rather than the SC which doesn't have the magic happening?)

Reply:
Coming in even later: yes, the `.html` should have technically been removed as well, but is ultimately harmless, as the output files actually reside at that location (and the canonical URLs without `.html` are effectively rewritten)